src/classes/EscalationTeamTrigger_Test.cls

```diff
@@ -0,0 +1,480 @@
+/**
+* Test for the escalation team trigger
+*
+* @author Preeti (ppreeti@redhat.com)
+*/
+
+@IsTest
+public with sharing class EscalationTeamTrigger_Test {
```

Comment:
Can you add some tests that show this working?  Like adding two team members that are not duplicates?  And updating a team member to something that is not a duplicate.